### ISSUE TITLE
bitcoin-abc-server-0.16.2 broken dependency

### ISSUE THREAD

**Issue by hippias** [error10/bitcoin-dist#29]
Hi, 

After trying to set up `bitcoin-abc-server`, it seems that there is a broken dependency definition:

```
sudo yum deplist bitcoin-abc-server
[...]
package: bitcoin-abc-server.x86_64 0.16.2-1.el7
  dependency: /bin/sh
   provider: bash.x86_64 4.2.46-29.el7_4
  dependency: /sbin/fixfiles
   provider: policycoreutils.x86_64 2.5-17.1.el7
  dependency: /sbin/restorecon
   provider: policycoreutils.x86_64 2.5-17.1.el7
  dependency: /usr/sbin/semodule
   provider: policycoreutils.x86_64 2.5-17.1.el7
  dependency: bitcoin-utils(x86-64) = 0.16.2                <------------------------ HERE
   Unsatisfied dependency
```

And the package does not seem to be present in the repo: https://www.ringingliberty.com/bitcoin/el7/x86_64/

When skipping the broken dependency the package does not get installed.

My server runs on CentOS 7.4.

Can you get this fixed or is there a way to help you?

PS: thanks for maintaining the repo and the associated documentation, it has been quite a relief for me :)

**Comment by error10** [comment]
Thanks for reporting this. Fixed builds are in progress and will be in the repos in a few hours.

**Comment by error10** [comment]
This should be fixed in 0.16.2-2.

**Comment by hippias** [comment]
Now it works like a charm, you made my day. Thanks!
